### DISCUSSION TITLE
README.md经常需要更新。是否支持在线编辑？

### DISCUSSION THREAD

**Discussion by tkfanswind** [spencerwooo/onedrive-vercel-index#81] [Ideas]
README.md经常需要。

现在只能下载改完，再上传。很麻烦
是否支持在线编辑

**Comment by spencerwooo** [comment]
你们 OneDrive 都不挂载本地吗？这东西难道不是本地直接改完 OneDrive 客户端就自动同步完了吗？

**Comment by tkfanswind** [comment]
还真没挂，25T太大了，怕同步到电脑上炸了 呵呵

**Reply by spencerwooo** [reply]
![image](https://user-images.githubusercontent.com/32114380/131647430-c0b23fb2-2e5d-498d-b716-da07e77982e7.png)

**Reply by opoet** [reply]
> 还真没挂，25T太大了，怕同步到电脑上炸了 呵呵

不取回本地是不占电脑空间的

**Comment by tkfanswind** [comment]
好。我试试

**Comment by tkfanswind** [comment]
如果能通过github里的一个文件在线编辑，实现某个目录的README.md这种功能就好了。直接在github上改
